tracer/build/_build/Build.GitHub.cs

```diff
@@ -359,8 +396,7 @@ partial class Build
 
             if (previousRelease is not null)
             {
-                sb.AppendLine($"[Changes since {previousRelease.Name}](https://github.com/DataDog/dd-trace-dotnet/compare/v{previousRelease.Name}...v{nextVersion})")
-                  .AppendLine();
+                sb.AppendLine($"[Changes since {previousRelease.Name}](https://github.com/DataDog/dd-trace-dotnet/compare/v{previousRelease.Name}...v{nextVersion})");
```

Comment:
If you trim above, we need to insert a line break beforehand
```suggestion
                sb.AppendLine()
                  .AppendLine($"[Changes since {previousRelease.Name}](https://github.com/DataDog/dd-trace-dotnet/compare/v{previousRelease.Name}...v{nextVersion})");
```

Reply:
Hmm, it's not the same. here the code you are modifying inserts an extra line in the release notes, after the issues. 
It' still relevant though so I'm adding it anyway in 670ff1e7ec78fd6da6c207a9d6798fa99b4f630e